packages/material-ui/src/Menu/Menu.js

```diff
@@ -43,24 +49,56 @@ const Menu = React.forwardRef(function Menu(props, ref) {
     classes,
     disableAutoFocusItem = false,
     MenuListProps = {},
+    // menuLevel = 1,
```

Comment:
remove

Reply:
Yup. Left it in because it's a super useful thing to log out to understand what level of the menu things are happening at. Doesn't seem like we'll need it for anything else now, so I'll pull it.
```suggestion
```